### DISCUSSION TITLE
Failed to minify the code from this file:       ./node_modules/react-hook-form/dist/index.esm.js:3

### DISCUSSION THREAD

**Discussion by BryanJoeHaboc** [react-hook-form/react-hook-form#6169] [Q&A]
Hello! I encountered a problem when building my react application 

![image](https://user-images.githubusercontent.com/69194216/128104148-eb359339-89c1-42a8-b94f-9323ba7fd258.png)

I had found similar issues but none of the solutions are working for me. I am using javascript. 

Here is my package.json file
{
  "name": "frontend-files",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@testing-library/jest-dom": "^5.11.4",
    "@testing-library/react": "^11.1.0",
    "@testing-library/user-event": "^12.1.10",
    "axios": "^0.21.1",
    "react": "^17.0.2",
    "react-datepicker": "^4.1.1",
    "react-dom": "^17.0.2",
    "react-hook-form": "^7.12.2",
    "react-onclickoutside": "^6.11.2",
    "react-router-dom": "^5.2.0",
    "react-scripts": "^1.1.5",
    "web-vitals": "^1.0.1"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": []
}

**Comment by BryanJoeHaboc** [comment]
You can see my repo here https://github.com/CantCodesheet/deployment-app-frontend . Many thanks!

**Comment by jorisre** [comment]
You have to use the last version of `react-scripts`.
You use the V1, the last version is 4.

You can find the migration guide in the official repository: https://github.com/facebook/create-react-app
